### ISSUE TITLE
Freeze when specific content changed

### ISSUE THREAD

**Issue by pionxzh** [yzhang-gh/vscode-markdown#465]
#### Environment
VS Code: 1.35.0
Markdown All in one: 2.3.1

#### What is the problem?
Today I just change `([example](https://example.com))`
to  `[Example]: https://example.com` and ` ( [Can I Use][Example] )` 
then boom~

If I change any content in the ` ( [Can I Use][Example] )` , the outline view will start to keep loading. And I can't delete any character until about 15 seconds freezing.

Is it an parsing issue  or ?

#### How can I reproduce it?

Markdown content: 
```markdown
# Example

[Example]: https://example.com

## Install

### Browser

SemanticApi is powered by `Proxy` ( [Can I Use][Example] ). Which is not available on Microsoft Internet Explorer or React Native (< v0.59) on An
```

Try edit the content in `Proxy ( [Can I Use][Example] ).`

#### Is there any error message in the console?

![example](https://i.imgur.com/DzhgD9G.png)

**Comment by yzhang-gh** [comment]
Thanks for the feedback.

I just tried your steps on my laptop. Everything looks fine.

Could you try again with all other extensions disabled? This is to see whether it is caused by any other extension.

![image](https://user-images.githubusercontent.com/7588612/59300658-a2202500-8c87-11e9-9c50-9ca30393cf97.png)

**Comment by pionxzh** [comment]
After disable all installed extension the problem disappeared.(Even though enable markdown all in one)

At the first second I found this problem, disabling markdown can drag me away from the freezing. Now it seems that there are some extension have conflict with it.

Thank, I will take some times to find out which extension cause the issue.

**Comment by tongyaop** [comment]
I have the same problem almost every time when I tried to type a link in markdown. 

- I disabled all other markdown-related extensions (incl. vim) but that did not solve the problem. I have to specifically disable markdown all in one to solve the problem. 
- I do not know if it is related to me working in a git repository (I have all the git-related extensions disabled. I tried to work in a different non-git test folder and the extension seems to be fine). 
- I am working on Windows
